spec/lib/site_setting_extension_spec.rb

```diff
@@ -857,6 +857,21 @@ def self.translate_names?
     end
   end
 
+  describe "mandatory_values for group list settings" do
```

Comment:
I guess this is the test to ensure the value is saved in the DB too

Reply:
คงมีประสิทธิภาพเพิ่มขึ้น